plugins/user-settings/src/components/ThemeToggle.test.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ApiProvider,
+  ApiRegistry,
+  appThemeApiRef,
+  AppThemeSelector,
+} from '@backstage/core';
+import { renderWithEffects, wrapInTestApp } from '@backstage/test-utils';
+import { lightTheme } from '@backstage/theme';
+import { fireEvent } from '@testing-library/react';
+import * as React from 'react';
+import { ThemeToggle } from './ThemeToggle';
+
+const mockTheme = {
+  id: 'light',
+  title: 'Mock Theme',
+  variant: 'light' as 'light', // wut?
+  theme: lightTheme,
+};
+
+const apiRegistry = ApiRegistry.from([
```

Comment:
Just a hint, the `.with()` variant of this is a slightly more convenient (and more type safe) alternative to `.from`